packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -195,7 +197,9 @@ export interface FieldEditor<TChangeset> {
  * The `index` represents the index of the child node in the input context.
  * The `index` should be `undefined` iff the child node does not exist in the input context (e.g., an inserted node).
  */
-export type ToDelta = (child: NodeId) => DeltaFieldMap;
+export type ToDelta = (
+	child: NodeId,
+) => [DeltaFieldMap, DeltaDetachedNodeChanges[], DeltaDetachedNodeRename[]] | undefined;
```

Comment:
Is there a reason this contract needs to change? (See the second paragraph in https://dev.azure.com/fluidframework/internal/_workitems/edit/13504)